### DISCUSSION TITLE
End to End communication on Cloud running NVIDIA ARC

### DISCUSSION THREAD

**Discussion by Tomwang78** [NVlabs/sionna#113] [Q&A]
Hi Expert,
 
Recently, we have plan to migrate our on premise end to end communication platform ( from Software define radio to Deep learning analytics platform ) to Cloud NVIDIA Aerial Research Cloud (ARC) running open source code Nvidia SIONNA to achieve higher compute resource efficiency.

Will like to know whether there is some good example or benchmarking code from  Sionna's benchmarking tools  that are designed to help me evaluate the performance of different deep learning algorithms eg Autoencoder or Reinforcement learning  and compare them with other implementations. 
Believe these tools are essential for assessing the efficiency and scalability of algorithms, which is crucial for developing future wireless networks.
For the benchmarking evaluation, I will consider measuring the execution time of algorithms, as well as memory usage and other performance metrics. This makes it easier to identify the most efficient algorithms and to optimize them for specific use cases.

I am thinking of generating a dataset from Software define radio - channel impairments and transmit signals using Sionna's simulation tools. Then, we can use  Autoencoder and reinforcement learning algorithm and other existing algorithms to equalize the received signals and measure the bit error rate (BER) performance of each algorithm using Sionna's BER measurement tools.
Next, we can use Sionna's benchmarking tools to compare the BER performance of the algorithm with other algorithms and generate performance reports. The benchmarking tools can provide insights into the strengths and weaknesses of each algorithm, as well as identify areas for improvement.
Thanks 
For your comments and i look forward for your generous reply 

Regard with a distance

**Comment by jhoydis** [comment]
Please contact the [Aerial Research Cloud](https://docs.nvidia.com/aerial/aerial-research-cloud/text/support.html) support for this.
